cocos/scene-graph/node.jsb.ts

```diff
@@ -1342,33 +1337,6 @@ nodeProto._ctor = function (name?: string) {
     this._euler = new Vec3();
 
     this._registeredNodeEventTypeMask = 0;
-
-    this.on(NodeEventType.CHILD_ADDED, (child) => {
-        this._children.push(child);
-    });
-
-    this.on(NodeEventType.CHILD_REMOVED, (child) => {
-        const removeAt = this._children.indexOf(child);
-        if (removeAt < 0) {
-            errorID(1633);
-            return;
-        }
-        this._children.splice(removeAt, 1);
```

Comment:
Move the logic to `_onChildAdded` `_onChildRemoved` to avoid adding two new listeners in JS.